app/src/main/java/ch/epfl/sdp/drone3d/model/mission/MappingMission.kt

```diff
@@ -21,14 +21,18 @@ enum class State {
 }
 
 /**
- * A MappingMission is instantiated with a [name], a [flightPath] and a [flightHeight].
+ * A MappingMission is instantiated with a [name], a [flightPath], a [flightHeight] and a [cameraPitch].
  * The [privateId], [sharedId] and [state] are updated according to where the MappingMission are stored.
  * The [ownerUid] is set the first time the mission is either stored or shared.
  */
 data class MappingMission(
+    //TODO: could be split into 2 substructure, one for the metadata and one for the
+    // actual mission(flight path, flight height, camera pitch)
+    // Only a shape should know be stored and the mapping mission regenerated for every drone
```

Comment:
I am positive about this statement, we should split it into two classes, one for the shape that is stored in the Database which can generate a mission with the drone information.